vite.config.js

```diff
@@ -33,5 +34,11 @@ export default defineConfig({
     minify: !process.env.TAURI_DEBUG ? 'esbuild' : false,
     // produce sourcemaps for debug builds
     sourcemap: !!process.env.TAURI_DEBUG,
+    rollupOptions: {
+      input: {
+        main: resolve(__dirname, 'index.html'),
+        nested: resolve(__dirname, 'src/preview/index.html'),
```

Comment:
Rename to `preview` to ensure output files match.
```suggestion
        preview: resolve(__dirname, 'src/preview/index.html'),
```